src/macro.internal.ts

```
@@ -0,0 +1,32 @@
+/**
+ * Prefetch claude data for the current user.
+ */
+
+import * as v from 'valibot';
+import { LITELLM_PRICING_URL } from './consts.internal.ts';
+import { type ModelPricing, ModelPricingSchema } from './pricing-fetcher.ts';
+
+// We'll fetch all claude models dynamically
+
+// eslint-disable-next-line antfu/no-top-level-await
+const response = await fetch(LITELLM_PRICING_URL);
```

ESLint

[error] Unsafe assignment of an error typed value.

(ts/no-unsafe-assignment)